app/assets/stylesheets/application.css.scss

```diff
@@ -255,8 +278,24 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
+    border-radius: 5px;
+    color: white;
+    background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces